projects/storefrontstyles/scss/components/layout/header/_header.scss

```diff
@@ -73,6 +73,7 @@ $space: 0.5rem;
   }
 
   .SiteLogo {
+    // display: block;
```

Comment:
I've just pushed a change here, which moves the focus related css rules to the banner component. that's where it belongs in my pov.